zenodo/modules/records/httpretty_mock.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DataCite.
```

Comment:
This header is probably from the `inveniosoftware/datacite` module. You can copy one from another Zenodo Python file.